src/Core/Commander/ManagerCommands.js

```diff
@@ -77,44 +80,35 @@ define('Core/Commander/ManagerCommands', [
             return this.commandsLength()===0;
         };
 
-        ManagerCommands.prototype.runAllCommands = function() {
-
-
-            if (this.commandsLength() === 0)
-            {
-                return Promise.resolve(0);
+        ManagerCommands.prototype.fillCommandPool = function() {
```

Comment:
`fillCommandPool` is describing *how* ManagerCommands does its job but fails to describe *what* this method is supposed to do.
IMHO `managerCommands.update()`, `managerCommands.execCommands()` are better. WDYT?